civis/parallel.py

```diff
@@ -32,6 +32,8 @@
         # models, you may need to re-serialize those models with
         # scikit-learn 0.21+."
         warnings.simplefilter('ignore', DeprecationWarning)
+        # sklearn 0.22 has switched from DeprecationWarning to FutureWarning
+        warnings.simplefilter('ignore', FutureWarning)
         from sklearn.externals.joblib import (
             register_parallel_backend as _sklearn_reg_para_backend)
         NO_SKLEARN = False
```

Comment:
could you add a comment like the following?

`# NO_SKLEARN_BACKEND would be a better name here since it’ll be true for future scikit-learn versions that won't include the joblib module as well as when scikit-learn isn’t installed.`